packages/examples/device_metrics/sensor_tag_ble_dell5k_mqtt_iotcc.py

```diff
@@ -0,0 +1,155 @@
+#      notice, this list of conditions and the following disclaimer in the    #
+#      documentation and/or other materials provided with the distribution.   #
+#                                                                             #
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"#
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE  #
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE #
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE  #
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR        #
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF       #
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS   #
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN    #
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)    #
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF     #
+#  THE POSSIBILITY OF SUCH DAMAGE.                                            #
+# ----------------------------------------------------------------------------#
+
+from liota.core.package_manager import LiotaPackage
+from liota.lib.utilities.utility import read_user_config
+
+dependencies = ["examples/mqtt/iotcc/iotcc_mqtt_edge_system_stats", "devices/sensor_tag"]
+
+
+def get_ambient_temperature(sensor):
+    tmp = sensor.get_temperature()
+    return tmp[0] if tmp else None
+
+
+def get_relative_humidity(sensor):
+    hum = sensor.get_humidity()
+    return hum[1] if hum else None
+
+
+def get_pressure(sensor):
+    # 1 millibar = 100 Pascal
+    pressure = sensor.get_barometer()
+    return (pressure[1] * 100) if pressure else None
+
+
+def get_battery_level(sensor):
+    return sensor.get_battery_level()
+
+
+def get_light_level(sensor):
+    return sensor.get_light_level()
+
+
+class PackageClass(LiotaPackage):
+    def run(self, registry):
+        import copy
+        import pint
+        from liota.entities.metrics.metric import Metric
+
+        # create a pint unit registry
+        ureg = pint.UnitRegistry()
+
+        # Acquire resources from registry
+        self.iotcc_edge_system = copy.copy(registry.get("iotcc_edge_system"))
+        self.iotcc = registry.get("iotcc_mqtt")
+        self.sensor_tag = registry.get("sensor_tag_device")
+
+        # Get values from configuration file
+        config_path = registry.get("package_conf")
+        self.config = read_user_config(config_path + '/sampleProp.conf')
+
+        # Registering SensorTagDevice with IoTCC
+        self.reg_sensor_tag = self.iotcc.register(self.sensor_tag)
+        self.iotcc.create_relationship(self.iotcc_edge_system, self.reg_sensor_tag)
+
+        # Create metrics
+        self.metrics = []
+        temperature_metric = Metric(
+            name="AmbientTemperature",
+            unit=ureg.degC,
+            interval=0,
```

Comment:
We provide sampling interval [here](https://github.com/Venkat2811/liota/blob/sensor-tag/examples/devices/sensor_tag_ble_dell5k_mqtt_iotcc.py#L167)
So, to collect from SensorTag, it must be zero.